### PR TITLE
IOPS 2258

### PR DESCRIPTION
**Pull request by KevinMayfield**
CapabilityStatement examples for discussion. 

These are not meant to be correct (the search parameters are known to be incorrect) and represent the current elaboration by programmes.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,8 +7,8 @@
   ],
   "dependencies": {
     "hl7.fhir.r4.core": "4.0.1",
-    "fhir.r4.ukcore.stu3.currentbuild": "0.0.8-pre-release",
```
Comment (APageNHS) [inline package.json:10]: What is the impact of removing this at the repo level - presumably NHSE IG package should bring in UK core via dependency for validation?

File: package.json [hunk 2]
```diff
@@ -7,8 +7,8 @@
...
-    "fhir.r4.nhsengland.stu1": "1.1.0"
+    "fhir.r4.nhsengland.stu1": "1.1.0",
+    "ihe.iti.pdqm": "2.4.0"
```
Comment (APageNHS) [inline package.json:11]: What is the benefit of adding this?

File: CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json [hunk 3]
```diff
@@ -0,0 +1,83 @@
+{
```
Comment (APageNHS) [inline CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json:1]: One of the things we're inconsistent on in IOPS is the file format, I was under the impression it should be XML?

File: package.json (hunk 1 above)
Comment (KevinMayfield) [reply package.json:10]: yes it does. Have tested with the validator. 

It's similar to the issue Kazeem mentioned where dependencies said 0.0.8 but  "fhir.r4.nhsengland.stu1": "1.1.0" which then changes the version back to 0.0.6

Comment (KevinMayfield) [reply package.json:10]: Happy to run through testing this today with simplifier but think need this approved to do that.

File: CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+  "resourceType": "CapabilityStatement",
+  "id": "England-NPS-Requirements",
```
Comment (APageNHS) [inline CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json:3]: QualityChecker is not picking up CapabilityStatements at, but filename / id/ url/ name / title are inconsistent and don't match NHSE IG Design  approach

File: package.json (hunk 2 above)
Comment (KevinMayfield) [reply package.json:11]: validator has been tested and no impact to other projects

File: CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json (hunk 3 above)
Comment (KevinMayfield) [reply CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json:1]: Yes, I want to use JSON and then bring this up at DA with a demo.

File: CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+      "coding": [
+        {
+          "system": "urn:iso:std:iso:3166:-2",
+          "code": "GB-ENG",
+          "display": "England"
+        }
+      ]
+    }
+  ],
+  "kind": "instance",
```
Comment (APageNHS) [inline CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json:36]: no purpose or copyright?

File: CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json (hunk 3 above)
Comment (KevinMayfield) [reply CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json:1]: with the reasons why XML is causing issues elsewhere.

File: CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+  "software": {
+    "name": "NHS England Validated Relationship Service"
+  },
+  "implementation": {
+    "description": "Test Instance"
+  },
+  "fhirVersion": "4.0.1",
+  "format": [
+    "application/fhir+json"
```
Comment (APageNHS) [inline CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json:45]: Is there a reason for not having xml?

File: CapabilityStatement/CapabilityStatement-England-PDS.json [hunk 7]
```diff
@@ -0,0 +1,149 @@
+{
+  "resourceType": "CapabilityStatement",
+  "id": "England-PDS-Instance",
+  "url": "https://fhir.nhs.uk/England/CapabilityStatement/England-Personal-Demograhics-Service",
+  "name": "NHSEnglandPersonalDemograhicsService",
+  "title": "Personal Demographics Service (FHIR) API",
+  "status": "unknown",
```
Comment (APageNHS) [inline CapabilityStatement/CapabilityStatement-England-PDS.json:7]: should this be drfat?

File: CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json (hunk 4 above)
Comment (KevinMayfield) [reply CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json:3]: what does that mean?

File: CapabilityStatement/CapabilityStatement-England-PDS.json (hunk 7 above)
Comment (APageNHS) [reply CapabilityStatement/CapabilityStatement-England-PDS.json:7]: or draft? ;)

File: CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json (hunk 6 above)
Comment (KevinMayfield) [reply CapabilityStatement/CapabilityStatement-England-National-Proxy-Service.json:45]: see above

File: CapabilityStatement/CapabilityStatement-England-PDS.json (hunk 7 above)
Comment (KevinMayfield) [reply CapabilityStatement/CapabilityStatement-England-PDS.json:7]: Don't know, I didn't create the original OAS

**Review by APageNHS — APPROVED** [review]
Approving on the grounds that KM needs this to bring to DA, resources will need to either be removed post DA decision, or reworked and brought in line with NHSE IG Design Approach for filenames / id / title / url / name / expected elements if these resources are being kept

**Pull request merged**